index.html

```diff
@@ -28,7 +28,7 @@
       content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0, shrink-to-fit=no"
     />
     <meta name="apple-mobile-web-app-capable" content="yes" />
-    <title></title>
+    <title>OpenMCT</title>
```

Comment:
Technically, 2 words:

```suggestion
    <title>Open MCT</title>
```

😄